subprojects/logging/src/testFixtures/groovy/org/gradle/internal/logging/text/TestStyledTextOutput.groovy

```diff
@@ -16,10 +16,9 @@
 
 package org.gradle.internal.logging.text
 
-import org.gradle.internal.SystemProperties
 import org.gradle.internal.logging.text.StyledTextOutput.Style
 
-class TestStyledTextOutput extends AbstractStyledTextOutput {
+class TestStyledTextOutput extends AbstractLineChoppingStyledTextOutput {
```

Comment:
Seems like there are many uses of `TestStyledTextOutput`. Is there any risk here changing the superclass, or should we consider creating a new `TestLineChoppingStyledTextOutput` instead of changing this class?